### PR TITLE
Improve `/addswipe` and `/delswipe` slightly

### PR DESCRIPTION
**Pull request by Wolfsblvt**
Minor additions to the swipe slash commands.
- `/addswipe` has an optional `switch` argument to go to the new swipe directly
- `/addswipe` returns the new swipe id
- `/delswipe` returns the swipe id you are currently on

Implements #2626

Was easily doable, so I just did it. Still think swipes are a core feature that could be better, and LA-Lib functions make much more sense for slash commands here.
We are missing a few things you should be able to do.

## Checklist:

- [x] I have read the [Contributing guidelines](https://www.youtube.com/watch?v=dQw4w9WgXcQ).

### CODE REVIEW

File: public/scripts/slash-commands.js [hunk 1]
```diff
@@ -2192,23 +2211,30 @@ async function addSwipeCallback(_, arg) {
...
             api: 'manual',
             model: 'slash command',
         },
     });
 
+    const newSwipeId = lastMessage.swipes.length - 1;
+
+    if (isTrueBoolean(args.switch)) {
+        lastMessage.swipe_id = newSwipeId;
+        lastMessage.mes = lastMessage.swipes[newSwipeId];
```
Comment (Cohee1207) [inline public/scripts/slash-commands.js:2231]: You forgor to switch `swipe_info`

Comment (Wolfsblvt) [reply public/scripts/slash-commands.js:2231]: Hmh? I don't get it. The `swipes` and `swipe_info` data gets created and added above. That was already there and I didn't touch it.

This if block is just for switching to this swipe. And I think to do that, you only need to replace the message text and set the swipe id.  
Nothing I can really do with the swipe_info here.

Comment (Cohee1207) [reply public/scripts/slash-commands.js:2231]: Makes sense. I seem to forgor of swipe_info details myself.

**Review by Cohee1207 — APPROVED** [review]
LGTM

**Pull request merged**